lib/ansible_tower_client/collection.rb

```diff
@@ -11,37 +11,47 @@ def all
     end
```

Comment:
@syncrou The refresh code on ManageIQ will call `.each` on the collections (now enumerators). https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/ansible_tower/configuration_manager/refresh_parser.rb#L50